rust/lance/src/dataset/scanner.rs

```diff
@@ -129,7 +133,11 @@ impl ColumnOrdering {
 pub struct Scanner {
     dataset: Arc<Dataset>,
 
-    projections: Schema,
+    // physical projections schema needed before final output
+    phyical_columns: Schema,
+
+    // projection of final output
+    output_expr: Option<Vec<(Expr, String)>>,
```

Comment:
Is this "all the output columns" or "additional columns on top of the physical schema"?  Can you clarify in the comment?